guides/source/testing.md

```diff
@@ -168,7 +168,7 @@ ArticleTest#test_should_not_save_article_without_title [/path/to/blog/test/model
 Expected true to be nil or false
 
 
-rails test test/models/article_test.rb:6
+bin/rails test test/models/article_test.rb:6
```

Comment:
This is the part of the output of the `bin/rails test` command executed above, which doesn't print `bin/rails`.

Reply:
Would it make sense to have it print `bin/rails`? Perhaps in a separate PR?

Reply:
Good idea 👍 We can probably make `bin/rails -T` print `bin/rails` instead of `rails` too.

Reply:
> `bin/rails -T` print `bin/rails`

@eugeneius @kaspth  I can make this happen, however I do have a question. 

User `cd` into `app` or elsewhere the command `rails -T` will bring up `bin/rails` as the user is no longer in the root of the app, `bin/rails` will no longer make sense, how do we handle that?

```
app HEAD % rails -T
bin/rails about                              # List versions of all Rails frameworks and t...
bin/rails action_mailbox:ingress:exim        # Relay an inbound email from Exim to Action ...
bin/rails action_mailbox:ingress:postfix     # Relay an inbound email from Postfix to Acti...
bin/rails action_mailbox:ingress:qmail       # Relay an inbound email from Qmail to Action...
bin/rails action_mailbox:install             # Installs Action Mailbox and its dependencies
```

Reply:
> We can probably make bin/rails -T print bin/rails

If we are in the root of the app and type the command `bin/rails -T` or `rails -T` the output would be legit i.e. bin/rails 

```
app HEAD % rails -T
bin/rails about                              # List versions of all Rails frameworks and t...
bin/rails action_mailbox:ingress:exim        # Relay an inbound email from Exim to Action ...
bin/rails action_mailbox:ingress:postfix     # Relay an inbound email from Postfix to Acti...
```

when we cd into another directory like app, we can run `rails -T` the output would still be the above, can we live with that?

I basically changed this line to `bin/rails` to get the output above in the line; https://github.com/rails/rails/blob/master/railties/lib/rails/commands/rake/rake_command.rb#L21

Reply:
I also updated my PR to print bin/rails for the test runner

Reply:
Both `rails test` and `rails -T` will have the problem you described when run from a subdirectory; we should change both or neither of them. Good catch, by the way!

Could we detect when the command was run with `bin/rails` and use it in the output in that case, otherwise fall back to `rails`?